wcivf/apps/people/models.py

```diff
@@ -206,19 +206,16 @@ def statement_count(self):
 
     @property
     def statement_intro(self):
-        statement_to_voters = self.statement_to_voters
-        statement_split = statement_to_voters.splitlines()
-        statement_intro = statement_split[0]
-
+        statement_intro = self.statement_to_voters.split(".")[0] + "."
         return statement_intro
 
     @property
     def statement_remainder(self):
-        statement_to_voters = self.statement_to_voters
-        statement_split = statement_to_voters.splitlines()
-        statement_intro = statement_split[0]
-        statement_remainder = statement_split.remove(statement_intro)
-        statement_remainder = "\n".join(statement_split)
+        statement_intro = self.statement_intro
+        # statement_to_voters = self.statement_to_voters
+        # statement_split = statement_to_voters.split("\n")
+        statement_split = self.statement_to_voters.split(".")
+        statement_remainder = "".join(statement_split[1:])
```

Comment:
I think this is a good improvement! We'll have to see how it works in the wild and might have to revisit the logic (I don't know how much I trust candidates to use English properly!). The good thing about the implementation is we can change it quickly if we need to!